src/helpers/keycard.ts

```diff
@@ -7,15 +7,18 @@ const keycard = new Keycard({
   URL: process.env.KEYCARD_URL || 'https://keycard.snapshot.org'
 });
 
-const verifyKeyCard = async (req, res, next) => {
-  const key = req.headers['x-api-key'] || req.query.apiKey;
-  if (key && keycard.configured) {
-    const { valid, rateLimited } = keycard.logReq(key);
+const checkKeycard = async (req, res, next) => {
+  const apiKey = req.headers['x-api-key'] || req.query.apiKey;
+  if (apiKey && keycard.configured) {
+    const keycardData = keycard.logReq(apiKey);
+    if (!keycardData.valid) return sendError(res, 'invalid api key', 401);
```

Comment:
Isn't this always returning `true` from keycard.js ? https://github.com/snapshot-labs/keycard.js/blob/f3bcc6a5b183bf02ba536127ac579656cae25d5d/src/index.ts#L77